packages/web-components-stencil/src/global/app.ts

```diff
@@ -0,0 +1,6 @@
+// import '@utrecht/web-component-library-stencil';
+import { defineCustomElements } from '@utrecht/web-component-library-stencil/loader';
```

Comment:
```suggestion
import { defineCustomElements as defineUtrechtCustomElements } from '@utrecht/web-component-library-stencil/loader';
```

Als we straks meer providers hebben zou het handig zijn om de naam van de provider te benoemen om duidelijkheid te creëeren.

Reply:
Ik zet dit iig ook bij de aanlever-documentatie waar ik momenteel mee bezig ben